charts/customer-center/templates/certificate-psql.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.postgresql.tls.enabled }}
+apiVersion: cert-manager.io/v1alpha3
+kind: Certificate
```

Comment:
I think there is something fishy about certs that "should" be part of a dependency but would need to dig deeper as to why other postgresql integrations (ie. timed) do not need this.

Reply:
I got this form the Timed chart, I just kept it. But the chart does not even let us the choice to create a ingress, which is ok right?

Reply:
Alright, got it. I am not sure about that, @fkm-adfinis do you know anything related to this?